lib/webhooks/deployment.payload.json

```diff
@@ -5,6 +5,7 @@
     "sha": "69a8b72e2d3d955075d47f03d902929dcaf74033",
     "ref": "master",
     "task": "deploy",
+    "name": "baxterthehacker/public-repo",
```

Comment:
Well, to tell you the truth I just assumed it was.
Heaven's test_fixtures had it and all the other attributes were replicated.
I just checked a recent one from a repo of mine and it's not actually there.
Now I'm confused. Whats the difference between the top level attributes and the inner objects?